R/automateDataPreparation.R

```diff
@@ -177,16 +178,19 @@ automateDataPreparation <- function(datList = NULL, inputList, path = NULL,
 
 		if(aggregateData) {
 			if(verbose) message("\nStart aggregating" )
-			if ( length(aggregatemissings) > 0 && aggregatemissings == "seeInputList" ) {
-				stopifnot(!is.null(inputList$aggrMiss))
-				aMiss <- unname(inputList$aggrMiss[,-1])
-				aMiss[,8] <- rep("err", 7)
-				aMiss[8,] <- rep("err", 8)
-				aggregatemissings <- unname(as.matrix(aMiss, nrow=8, ncol=8))
-				dimnames(aggregatemissings) <-
-                list(c(names(inputList$aggrMiss)[-1], "err"),
-										c(names(inputList$aggrMiss)[-1], "err"))
-			}
+
+		#	if ( length(aggregatemissings) > 0 && aggregatemissings == "seeInputList" ) {
+			if (!is.null(inputList$aggrMiss)) {
```

Comment:
`inputList$aggrMiss` looks a little strange: 
```
  c..vc....mvi....mnr....mci....mbd....mir....mbi..  vc mbd mvi mnr mci mir mbi
1                                                vc  vc err mvi  vc mci  vc  vc
2                                               mvi mvi err mvi err mci err err
3                                               mnr  vc err err mnr mci mir mnr
4                                               mci mci err mci mci mci mci mci
5                                               mbd err mbd err err err err err
6                                               mir  vc err err mir mci mir mir
7                                               mbi  vc err err mnr mci mir mbi
```

Reply:
Yes, but this is the standard output given by the EDV programmed tool. In the standard case, people shouldn't have to modify this by hand. That's why I chose this 'strange' format as the standard in data(inputList).

Reply:
All right, only the first column's name is strange. But you can leave it as it is.